nomos-services/mix/src/lib.rs

```diff
@@ -77,12 +81,41 @@ where
             mut backend,
             network_relay,
         } = self;
+        let mix_config = service_state.settings_reader.get_updated_settings();
 
         let network_relay = network_relay.connect().await?;
         let network_adapter = Network::new(network_relay);
 
-        // TODO: Spawn PersistentTransmission (Tier 1)
-        // TODO: Spawn Processor (Tier 2) and connect it to PersistentTransmission
+        // Spawn Persistent Transmission
+        let (transmission_schedule_sender, transmission_schedule_receiver) =
+            mpsc::unbounded_channel();
+        let (emission_sender, mut emission_receiver) = mpsc::unbounded_channel();
+        tokio::spawn(async move {
+            persistent_transmission(
+                mix_config.persistent_transmission,
+                transmission_schedule_receiver,
+                emission_sender,
+            )
+            .await;
+        });
```

Comment:
Yep, this is another signal that it could probably be implemented as stream operations. Lets keep this in mind for a further refactor. Maybe open an issue for it.

Reply:
https://github.com/logos-co/nomos-node/pull/844#discussion_r1816233929